03-Node-npm-modules/js/lib/movie.js

```diff
@@ -0,0 +1,50 @@
+//LIBS
+
+var Director = require('./director.js');
+var $ = require('jquery');
+
+//FUNCTIONS
+
+var MovieObserver = function ( ) {};
+MovieObserver.prototype.playing = function (title) {
+    console.log('A movie with title ' + title + ' has been played');
+};
+MovieObserver.prototype.stopped = function(title) {
+    console.log('A movie with title ' + title + ' stopped');
+};
+
+var Movie = function (title) {
+  this.title = title;
+  this.duration = ''; 
+  this.director = new Director('');
+  this.observer = new MovieObserver();
+};
+
+Movie.prototype = {
+  constructor : Movie,
+  get : function (property) {
+    return this[property];
+  }, 
+  set :function (property, value) {
+    this[property] = value; 
+  },
+  play :function () {
+    console.log('This is Movie.play method: I am a movie. This is my title: ' + this.title);
+    this.observer.playing(this.title);
+  },
+  stop :function () {
+    this.observer.stopped(this.title);
+  }
+};
+
+var alien = new Movie('Alien');
+var ridleyScott = new Director('Ridley Scott');
+ridleyScott.set('quotes', ['Cast is everything.', 'Do what ...']);
+alien.set('Director', ridleyScott);
+console.log(alien.get('Director').speak());//output: Ridley Scott says: 'Cast is...'
+
+$(document).ready(function() {
```

Comment:
Good work! Everything looks pretty good, the only thing I would do is to extract all implementations (starting from line 39) into another file and leave this as the class definition. :)